src/Action/RefundOrderAction.php

```diff
@@ -49,22 +49,27 @@ public function execute($request): void
 
         $details = ArrayObject::ensureArrayObject($request->getModel());
 
-        if ($details['created_in_mollie']) {
-            $this->loggerAction->addLog('Received refund created in Mollie dashboard');
+        try {
+            $order = $this->mollieApiClient->orders->get($details['order_mollie_id'], ['embed' => 'payments']);
+            $payments = $order->_embedded->payments;
+            $payment = current($payments);
```

Comment:
There will be always one payment?

Reply:
No, I will fix it to take only 'paid' payment